docs/reference/installation.rst

```diff
@@ -160,47 +122,17 @@ With provided parameters, the files are generated in ``src/Application/Sonata/Co
     point to a global namespace. For instance the user will be
     ``App\Application\Sonata\CommentBundle\Entity\Thread``.
 
-.. note::
-    If you are not using Symfony Flex, the namespace will be ``Application\Sonata\CommentBundle\Entity``.
-
-Now, add the new ``Application`` Bundle into the ``bundles.php``:
-
-.. code-block:: php
-
-    <?php
+Now, add the new ``Application`` Bundle into the ``bundles.php``::
 
     // config/bundles.php
 
     return [
-        //...
+        // ...
         App\Application\Sonata\CommentBundle\ApplicationSonataCommentBundle::class => ['all' => true],
     ];
```

Comment:
Do we have an AppBundle?

Reply:
What do you mean? 🤔

Reply:
There is no `ApplicationSonataCommentBundle`

Reply:
This namespace looks wrong, the name should be something like this:

`App\SonataCommentBundle\AppSonataCommentBundle::class`

or

Reply:
I agree, but it gets genrated by the easy extends command...

Reply:
I know, but we don't need this anymore :thinking:

Reply:
Yes, but we don't want to maintain easy-extends anymore. And besides that, I don't think this bundle is used so much, so lets keep it like this and merge it until someone reports some issues. Are you ok with that?